mypy/checkexpr.py

```diff
@@ -4281,6 +4284,11 @@ def has_any_type(
     return t.accept(HasAnyType(ignore_in_type_obj, ignore_any_from_error=ignore_any_from_error))
 
 
+def has_untyped_type(t: Type, ignore_in_type_obj: bool = False) -> bool:
+    """Whether t contains an untyped type"""
```

Comment:
NIT (full form: Not Important Todo):
```suggestion
    """Whether ``t`` contains an untyped type"""
```